vertx-web/src/main/java/io/vertx/ext/web/handler/impl/ErrorHandlerImpl.java

```diff
@@ -95,9 +92,11 @@ private boolean answerWithKnownFailure(RoutingContext context) {
     
     if(failure instanceof HeaderTooLongException){
       // Math.random is random enough for this purpose. Will wait between 1 ms and 11ms
-      answerDelayTimer.schedule(
-          new DelayedAnswer(context, 400, "A header was too long to process"), 1 + (long)(Math.random() * 10)
+      
```

Comment:
This is not timeout handling. This is work faking.
If one tries to attack the server and sees that if a certain action done will make the server take longer to respond, that attacker will think that the server is working during all that time.

If you don't want such thing and prefer it to answer immediately, fine by me. I just thought this would be good to make any attacker interested trying to DDoS to use a less effective methods because they takes longer to respond than the most effective ones.